lib/verification/verify.go

```diff
@@ -63,6 +64,10 @@ func (v *VerificationModule) VerifyHAAWSManagedProvision(configFile string) erro
 	if err != nil {
 		return err
 	}
+	configVerify := &config_verify.ConfigVerifyImpl{}
```

Comment:
line num 67 to 70 & 81 to 84 & 95 to 98  are duplicates. You can write a common method

Reply:
updated code.